data/campaigns/Eastern_Invasion/scenarios/13_Spoils_of_War.cfg

```diff
@@ -584,6 +588,8 @@
         {MOVE_UNIT side=1 42 4}
         [message]
             id=Owaec,Terraent
+            #po: TODO for 1.19? There's a cut-scene running up to this, but it's unclear what's happened until I read the WML and found that player's full starting roster is exactly one *horseman* for this scenario.
+            #po: The fastest-moving of the player's units has been trying to catch up with a bat, but the bat has flow over the river to the Mal-Yrna / Mal-Mana.
```

Comment:
Typo: has flown

Also, why are you using an article on a proper name?

Reply:
Both points are typos, will fix in 1.19.